synapse/models/telco.py

```diff
@@ -104,16 +182,57 @@ def initCoreModule(self):
     def getBaseModels():
         modl = {
             'types': (
+
                 ('tel:phone', {'ctor': 'synapse.models.telco.PhoneType'}),
+
+                ('tel:mob:tac', {'subof': 'int',
+                                 'doc': 'A mobile Type Allocation Code'}),
+
+                ('tel:mob:imei', {'ctor': 'synapse.models.telco.ImeiType',
+                                  'doc': 'An International Mobile Equipment Id'}),
+
+                ('tel:mob:imsi', {'ctor': 'synapse.models.telco.ImsiType',
+                                  'doc': 'An International Mobile Subscriber Id'}),
+
+                # TODO: mcc, meid
+
             ),
+
             'forms': (
+
                 ('tel:phone', {'ptype': 'tel:phone'}, [
                     ('cc', {'ptype': 'pol:iso2', 'defval': '??'}),
                 ]),
+
                 ('tel:prefix', {'ptype': 'tel:phone'}, [
                     ('cc', {'ptype': 'pol:iso2', 'defval': '??'}),
                     ('tag', {'ptype': 'syn:tag'}),
                 ]),
+
+                ('tel:mob:tac', {}, [
+
+                    ('org', {'ptype': 'ou:org', 'defval': 'ffffffffffffffffffffffffffffffff',
```

Comment:
if we want a ``ffffffffffffffffffffffffffffffff`` ou:org node we should probably make that in the init of the Org coremodule and set its props to note it is a default gravity well org that isn't analytically useful